tests/tfengine/integration_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfengine"
+)
+
+var (
+	dirs = []string{"project_secrets"}
+
+	confTmpl = template.Must(template.New("").Parse(`
```

Comment:
I would move this to a function called "buildTmpl" that can be called next to buildData. That way the two are close together, which is beneficial since one uses the other.